libraries/src/Menu/AbstractMenu.php

```diff
@@ -364,4 +389,43 @@ public function load()
 	{
 		return array();
 	}
+
+	/**
+	 * Internal getter for the user. Returns the injected
+	 * one if any, or the current one if none.
+	 *
+	 * @return User
+	 *
+	 * @since 3.24.0
```

Comment:
That since tag should be 3.9.23 right?

Reply:
```suggestion
	 * @since __DEPLOY_VERSION__
```

or `__DEPLOY_VERSION__` to be sure ;)